src/vmm/src/guest_config/templates/test_utils/mod.rs

```diff
@@ -0,0 +1,227 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(target_arch = "x86_64")]
+use crate::guest_config::cpuid::KvmCpuidFlags;
+#[cfg(target_arch = "aarch64")]
+use crate::guest_config::templates::aarch64::{RegisterModifier, RegisterValueFilter};
+#[cfg(target_arch = "x86_64")]
+use crate::guest_config::templates::x86_64::{
+    CpuidLeafModifier, CpuidRegister, CpuidRegisterModifier, RegisterModifier, RegisterValueFilter,
+};
+use crate::guest_config::templates::CustomCpuTemplate;
+
+/// Create architecture agnostic type aliases
+
+/// Test CPU template in JSON format
+#[cfg(target_arch = "aarch64")]
+pub const TEST_TEMPLATE_JSON: &str = r#"{
```

Comment:
would it be too much to separate these into arch-specific files?

Reply:
Yea this is fair. I think this should be easy enough while maintaining the namespace path.